boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpurad.dts

```diff
@@ -34,10 +34,6 @@
 	};
 };
 
-&shared_ram3x_region {
```

Comment:
And this with `&cpurad_dma_region`?

Reply:
`cpurad_dma_region` is already enabled further down.

Reply:
Missed this, thanks!